src/styles/block/card.css

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
```

Comment:
the card width size is bigger than 200px, you can fix it by using the box-sizing property for *